node/config/def.go

```diff
@@ -170,6 +170,9 @@ func DefaultBoost() *Boost {
 			MaxDealsPerPublishMsg: 8,
 			MaxPublishDealsFee:    types.MustParseFIL("0.05"),
 		},
+		Curio: CurioConfig{
+			Enabled: false,
+		},
```

Comment:
Ack, I will move this to top. This was just temporary for testing. Makes easy to add at the end for testing.

Reply:
I tried this. This can be done for all use cases except libp2p userAgent. That part is initialised well before any other. If you have any approach to do this, let me know. I will also explore some other ideas.